superset/assets/javascripts/SqlLab/components/VisualizeModal.jsx

```diff
@@ -12,10 +12,10 @@ import { getExploreUrl } from '../../explore/exploreUtils';
 import * as actions from '../actions';
 
 const CHART_TYPES = [
-  { value: 'dist_bar', label: 'Distribution - Bar Chart', requiresTime: false },
-  { value: 'pie', label: 'Pie Chart', requiresTime: false },
-  { value: 'line', label: 'Time Series - Line Chart', requiresTime: true },
-  { value: 'bar', label: 'Time Series - Bar Chart', requiresTime: true },
+  { value: 'dist_bar', label: 'Distribution - Bar Chart', requiresTime: false, requiresMatrix: true },
```

Comment:
rather than calling this `requiresMatrix`, would it be more clear to specify `requiresDimension` and `requiresAggregrateFn`?